cypress/integration/common/i_pause.js

```diff
@@ -0,0 +1,7 @@
+import { Then } from "cypress-cucumber-preprocessor/steps";
+
+// Pause.  This is not recommended best practice but due to Cypress being too quick at doing certain things
+// we do this in (minimal possible) places until we have fixed underlying issues
+Then('I pause', () => {
```

Comment:
Still don't like this. Is there any way to just keep checking the number of elements in the drop-down until it "settles" and has the correct list in it? Alternatively, is it possible to blat the search term into the box without typing it, so the lookahead doesn't start populating with a partial postcode? (is that what force fill does?)

Reply:
This is 100% a temp fix. Its either, leave this in for now or take it out and tests will currently fail. The intention of the wait is to make sure Cypress finished typing in the postcode, before hitting the button. I intend to find a way for Cypress to double check its finished typing. i:e re-inspect the box to see if it contains what Cypress should have typed in. Feels a bit wrong though. Cypress shouldn't be behaving like this. Am working on a long-term solution is basically the answer to this.

Reply:
Would it make sense to raise a bug specifically for this? Is it not possible to force the value on the element, or do you have to type for the lookahead to work?

Reply:
OK, I now replaced the i_pause with an explicit check that the element is filled in. This should keep trying.  (force doesn't apply to this situation I don't believe - the issue seems to simply be cypress hitting the lookup button before finishing typing)